### DISCUSSION TITLE
A small doubt about mob spawning with this mod

### DISCUSSION THREAD

**Discussion by TechPro424** [vlad2305m/fabric-per-player-spawns#8] [Q&A]
Ok, so if there were 2 players extremely far away from each other, they would each have their own mob cap? (Like, each player would have 70 hostile mobs, 10 passive mobs, etc)

**Comment by vlad2305m** [comment]
Yes, this mod would limit mobs around each player by 1 mob cap, however, if there are more mobs around one of the players for any reason, it would bypass the original mobcaps and still give the other player full mobcaps for each spawn group.
